contracts/v4/Core.sol

```diff
@@ -0,0 +1,269 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.19;
+
+interface Core {
```

Comment:
this is outdated, can you please have a look at https://github.com/immutable/contracts/tree/main/contracts/bridge/x/v4